python/ray/_private/runtime_env/agent/main.py

```diff
@@ -211,6 +211,7 @@ def parent_dead_callback():
             host=args.node_ip_address,
             port=args.runtime_env_agent_port,
             loop=loop,
+            keepalive_timeout=None,  # we want to always keep alive.
```

Comment:
Hmm actually I am confused with this comment. If keepalive_timeout = None, does that mean keepalive will never timeout, meaning it can actually never detect the client side failures?